apps/api-gateway/src/public-data/public-data.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { PublicDataService } from './public-data.service';
+
+@Controller('pub-data')
+@ApiTags('Public Data API')
+export class PublicDataController {
+  public constructor(private readonly publicDataService: PublicDataService) {}
+
+  @Get('post')
+  @ApiOperation({ summary: '전체 post category 데이터 조회' })
+  @ApiCreatedResponse({
+    description: '전체 post category 데이터 조회',
+  })
+  public async getPostData() {
+    const postDatas = await this.publicDataService.getPostData();
+    return postDatas;
```

Comment:
임시변수를 두지 않고 이렇게 작성하는 것과는 어떤 차이가 있을까요?
```suggestion
    return await this.publicDataService.getPostData();
```

Reply:
확실히 단축해서 작성하는게 훨씬 낫겠네요...
습관적으로 변수화해서 작성했었는데 위의 방식대로 수정 후
커밋으로 반영 하여 추가 코멘트 올려보겠습니다!

Reply:
다른 내용이지만 이 글도 한번 읽어보시면 좋을 것 같네요!
https://jojoldu.tistory.com/699